Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -52,10 +46,26 @@ static DWRITE_TEXT_ALIGNMENT __CoreTextAlignmentToDwrite(CTTextAlignment alignme
     }
 }
 
+static DWRITE_WORD_WRAPPING __CTLineBreakModeToDWrite(CTLineBreakMode lineBreakMode) {
```

Comment:
both, I think? we have unit tests that test _ functions

---
In reply to: [90918451](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90918451) [](ancestors = 90918451,90746929,90742638)